cl/assets/static-global/css/override.css

```diff
@@ -1837,3 +1837,7 @@ rect.series-segment {
 .gap-2 {
   gap: 0.5rem
 }
+
+.prayer-top-buttons > li > a {
+  padding: 10px 15px !important;
```

Comment:
We usually try hard not to use `!important`, preferring instead to use specificity to win CSS wars. Does this fail without the `important` argument?

Reply:
My apologies, I was aware of that 🥲.

I've just found a solution that avoids the use of !important.